RibonDAO/core-api
# Feat/count today donations

# Description

Adds a route that counts all donations made today, and caches the value for an hour

## Type of change

Please delete options that are not relevant.


## How to test

- Change the expiration time to 30 seconds
- Make an get request to '/api/v1/count_today_donations'
- Check if it used the cache only after the first request
